docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Ibexa Engage helps to engage audiences and collect valuable data by using interactive content.
+---
+
+# Ibexa Engage
+
+[[= product_name_engage =]] is a data collection tool. It gives you the ability to use the [Qualifio](https://qualifio.com/) tools to engage your audiences.
+You can use interactive content to build relationships and collect important data, for example, a list of recent orders, or personal information about customers.
+
+## Request access
+
+To use [[= product_name_engage =]], you must make arrangements with [[= product_name =]] to define the initial configuration, and then get and set up user account.
+
+1\. Go to the Back Office.
+
+2\. On the left panel, go to **Ibexa Engage**.
+
+3\. On the welcome page, provide the required information:
+
+- First and last name
+- Email address
+
+4\. Select the **I have read and agree to the Terms and Conditions** checkbox, and then click **Send request**.
+
+![Welcome page - form](img/welcome_page_form.png "Welcome page - form")
+
+[[= product_name =]] creates and provides user account. An invitation link is sent during the setup process.
```

Comment:
"Ibexa DXP creates and provides user account. An invitation link is sent during the setup process."-  this sentence sounds for me like the process would be automatic. And it's not at the moment. It's manual and credentials are sent by our team, right @SylvainGuittard ?
Maybe it's a matter of changing the word from Ibexa DXP to Ibexa team?

Reply:
Changed to _Ibexa team_.